staff/daniel-villegas/cv/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>The CV</title>
+</head>
+<body>
+    <aside>
```

Comment:
Are you sure `<aside>` is the right html tag here?
[https://developer.mozilla.org/en-US/docs/Web/HTML/Element/aside](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/aside)

Reply:
True, I though was for englobe the sections :P

Changed!